File: sumArrays.java
public class Main {
	public static void main(String[] args) {
		int array[] = {3,7,9,20,58};
		int result = 0;
		for(int i = 0; i < array.length; i++){
			System.out.println(i + ": " + array[i]);
			result += array[i];
		}
		System.out.println(result);
	}
}

File: sumArrays.cpp
#include <bits/stdc++.h>
#include <iostream>

using namespace std;
int main() {
	int array[] = {2,6,9,3,5};
	int result = 0;
	for(int i = 0; i < sizeof(array)/sizeof(int); i++){
		cout << i << ": " << array[i] << endl;
		result += array[i];
	}

	cout << result << endl;
	return 0;
}
